services/base-service/models/expert/api.js

```diff
@@ -23,8 +24,22 @@ function subselect(req, res, next) {
     let expertId = `${req.params.expertId}`;
     params.admin = req.user?.roles?.includes('admin') || expertId === req?.user?.attributes?.expertId;
 
-    res.thisDoc = model.subselect(res.thisDoc, params);
-    next();
+    // Here, in the API, we might require different authentication
+
+    if (params?.grants?.size > 10 || params?.works?.size > 10) {
+      // We verify they are logged into CAS
+      is_user(req, res, (err) => {
```

Comment:
We'll want to make sure this works correctly when download an experts works/grants.

Reply:
@qjhart 
I think the `subselect` function in the experts model, by default, will include all results, if the expert/grant/work has `include:true` passed in. So size is optional. 

Perhaps we should adjust this to check something like this?-
`params?.grants?.include && (!params.grants.size || params.grants.size > 10)`  

Or do you think we should change the model.subselect so that it doesn't return everything by default?